### PR TITLE
Geoms

### PR DESCRIPTION
**Pull request by maxwellfundi**

### CODE REVIEW

**Comment by maxwellfundi** [comment]
@dannyparsons This control still has lots to be worked on. I wonder how i would pass parameters from the ucrreceiver params to aes function of the geom

File: instat/UcrGeomListWithAes.designer.vb [hunk 1]
```diff
@@ -0,0 +1,232 @@
+﻿<Global.Microsoft.VisualBasic.CompilerServices.DesignerGenerated()> _
+Partial Class UcrGeomListWithParameters
```
Comment (dannyparsons) [inline instat/UcrGeomListWithAes.designer.vb]: This should have lower case "u"

File: instat/UcrGeomWithAes.designer.vb [hunk 2]
```diff
@@ -0,0 +1,98 @@
+﻿<Global.Microsoft.VisualBasic.CompilerServices.DesignerGenerated()> _
+Partial Class UcrGeomWithAes
```
Comment (dannyparsons) [inline instat/UcrGeomWithAes.designer.vb]: lower case "u" needed

File: instat/UcrGeomWithAes.designer.vb [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    'NOTE: The following procedure is required by the Windows Form Designer
+    'It can be modified using the Windows Form Designer.  
+    'Do not modify it using the code editor.
+    <System.Diagnostics.DebuggerStepThrough()> _
+    Private Sub InitializeComponent()
+        Me.TextBox1 = New System.Windows.Forms.TextBox()
+        Me.TextBox2 = New System.Windows.Forms.TextBox()
+        Me.TextBox3 = New System.Windows.Forms.TextBox()
+        Me.TextBox4 = New System.Windows.Forms.TextBox()
+        Me.TextBox5 = New System.Windows.Forms.TextBox()
```
Comment (dannyparsons) [inline instat/UcrGeomWithAes.designer.vb]: textboxes need correct names

File: instat/dlgGeneralForGraphics.vb [hunk 4]
```diff
@@ -15,8 +15,19 @@
...
+    Private clsRgeom_Function As New RFunction
+    Private clsRggplotFunction As New RFunction
+    Private clsRaesFunction As New RFunction
+
+    Private bFirstLoad As Boolean = True
+    Public clsRSyntax As New RSyntax
+    Private Sub dlgGeneralForGraphics_Load(sender As Object, e As EventArgs) Handles MyBase.Load
+        clsrgeom_function.setrcommand(UcrGeomListWithParameters1.cboGeomList.SelectedItem)
+        clsraesfunction.setrcommand("aes")
+        clsrgeom_function.addparameter("mapping", clsrfunctionparameter:=clsraesfunction)
```
Comment (dannyparsons) [inline instat/dlgGeneralForGraphics.vb]: Not sure why these don't have the correct case? Maybe check in VS.

File: instat/clsGeom.vb [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+' (at your option) any later version.
+'
+' This program is distributed in the hope that it will be useful,
+' but WITHOUT ANY WARRANTY; without even the implied warranty of
+' MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+' GNU General Public License for more details.
+'
+' You should have received a copy of the GNU General Public License k
+' along with this program.  If not, see <http://www.gnu.org/licenses/>.
+Public Class Geoms
```
Comment (dannyparsons) [inline instat/clsGeom.vb]: "Geom" would be a better name, since each instance of this is a single geom

File: instat/UcrGeoms.vb [hunk 6]
```diff
@@ -0,0 +1,87 @@
+﻿Public Class UcrGeoms
```
Comment (dannyparsons) [inline instat/UcrGeoms.vb]: lower case "u" needed

File: instat/UcrGeomWithAes.designer.vb (hunk 3 above)
Comment (maxwellfundi) [inline instat/UcrGeomWithAes.designer.vb]: I removed these already when i removed the unused control

File: instat/dlgGeneralForGraphics.vb (hunk 4 above)
Comment (maxwellfundi) [inline instat/dlgGeneralForGraphics.vb]: what do you mean?

**Comment by maxwellfundi** [comment]
@dannyparsons  i have made the additions, tested the aes and is working well

**Comment by maxwellfundi** [comment]
@dannyparsons  this is cool for merging, have a look

**Pull request merged**